[Project v1.1] Update About Us documentation

QzSG opened the pull request:
Resolves #25

QzSG committed:
Update AboutUs.adoc with my details (#26)

* Update info in AboutUs.adoc

Add photo of me
Update information of me in About us

ChenSongJian committed:
Updated info in AboutUs.adoc

ChenSongJian committed:
Updated AboutUs.adoc

T0roloco committed:
Update AboutUs.adoc and added images (#27)

* Update AboutUs.adoc and added images

* Fix Mario photo url

luhan02 committed:
Update AboutUs.adoc

ian-tjahjono committed:
Changed AboutUs and added picture (#30)

GSzQ reviewed (changes requested):
Missing responsibilities for Lu Han

QzSG committed:
Fix AboutUs.adoc

Remove portfolio links for now
Ensure everyone has a responsibility (not finalised)
